src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -761,10 +761,12 @@ private SmugglerProgressBase.Counts ProcessCompareExchange(SmugglerResult result
             using (_database.DocumentsStorage.ContextPool.AllocateOperationContext(out JsonOperationContext context))
             using (var actions = _destination.CompareExchange(context))
             {
-                foreach (var kvp in _source.GetCompareExchangeValues())
+                foreach (var kvp in _source.GetCompareExchangeValues(actions))
                 {
```

Comment:
Need to dispose the `StorageKey` as well

Reply:
the key is string